src/components/RemediationTable.js

```diff
@@ -168,10 +175,22 @@ function RemediationTable({
                 dispatch(getEndpoint(id));
               }}
               sources={sources}
+              activeAlert={activeToastAlert}
+              setActiveAlert={setActiveToastAlert}
             />
           )}
         </StackItem>
         <StackItem>
+          {activeToastAlert.title !== '' ? (
+            <PlaybookToastAlerts
+              key={activeToastAlert.key}
+              title={activeToastAlert.title}
+              description={activeToastAlert.description}
+              variant={activeToastAlert.variant}
+            />
+          ) : (
+            <> </>
```

Comment:
If you want to return an empty element, return a `null` not fragment. It just adds an extra node to the virtualDOM.

You could also use the `{stuff && <Component />}` instead of the ternary operator.